src/EventStore.Core/Services/Transport/Grpc/RpcExceptions.cs

```diff
@@ -76,7 +76,7 @@ public static class RpcExceptions {
 			new(
 				new Status(
 					StatusCode.FailedPrecondition,
-					$"Append failed due to WrongExpectedVersion. Stream: {streamName}, Expected version: {expectedVersion}, Actual version: {actualVersion}"),
+					$"Delete failed due to WrongExpectedVersion. Stream: {streamName}, Expected version: {expectedVersion}, Actual version: {actualVersion}"),
```

Comment:
Or just change the message to `Operation` so that it works for every case:

```suggestion
					$"Operation failed due to WrongExpectedVersion. Stream: {streamName}, Expected version: {expectedVersion}, Actual version: {actualVersion}"),
```